src/parser/TurtleParser.h

```diff
@@ -205,8 +202,8 @@ class TurtleParser {
   }
 
   // create a new, unused, unique blank node string
-  string createBlankNode() {
-    string res = "_:" + std::to_string(_numBlankNodes);
+  string createAnonNode() {
+    string res = ANON_NODE_PREFIX + ":" + std::to_string(_numBlankNodes);
     _numBlankNodes++;
```

Comment:
Any reason this is now called "Anon" instead of "Blank". Also we are parsing in parallel now, right? So what prevents `_numBlankNodes++` updates from racing?

Reply:
1. This is only used for Anon nodes now: Those must be unique, so we need to find our own representation with the counter + some prefix that never occurs in any knowledge base.
Previously I also did this for blank nodes, but that slowed down the parser.
Blank nodes can be repeated and we have to make sure that the same blank node in the input stands for the same blank node in the graph/internal representation. For this we can simply reuse the additional blank node. So I renamed this, since it is only used for unique anon nodes.

2. There is only one parser thread and the parser itself is not threadsafe (there is no real good way to parse in parallel with LL1-Grammars. There could be a parallel parsing of Wikidata using their dumps, under the following assumptions:
- All prefix/base declarations come before all triples.
- There is a unique way to determine the beginning of a statement (e.g. newlines that are not followed by whitespace).

I even thought about recommending this to the W3 community, as it would make the parsing so so much faster. But we are currently getting closer to the disk write being our limiting factor.
But give me parallel disks and there is much more that we can speed up.

So currently: parser not threadsafe.